evap/staff/templates/staff_user_form.html

```diff
@@ -16,6 +16,14 @@
     {{ block.super }}
     <h3>{% if form.instance.id %}{% trans 'Edit user' %}{% else %}{% trans 'Create user' %}{% endif %}</h3>
 
+    <div class="card card-submit-area text-center mb-3">
```

Comment:
I don't think this is exactly what the issue meant with "top right corner". It should probably look closer to what the "Export results" button looks like on the contributor index (in non-staff mode)